src/backend/distributed/planner/relation_restriction_equivalence.c

```diff
@@ -59,6 +59,14 @@ typedef struct AttributeEquivalenceClass
 	Index unionQueryPartitionKeyIndex;
 } AttributeEquivalenceClass;
 
+typedef struct FindRteIdentityContext
```

Comment:
the name sounds hard to understand. Maybe `FindQueryContainingRteIdentityContext`